src/sphinx_book_theme/assets/styles/base/_base.scss

```diff
@@ -10,6 +10,10 @@
   left: 0;
 }
 
+.d-none {
+  display: none;
+}
+
```

Comment:
```suggestion
```

My point was that I don't think we need this at all, because bootstrap should already give it to us

Reply:
aaah yeah. ofcourse.